plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -187,6 +151,7 @@ public Version getVersion() throws IOException {
      * @return list of docker images
      * @throws IOException
      */
+    @SuppressWarnings("unused")
```

Comment:
I don't think that it is the place for this suppression, as it is API method. I think you should suppress 'unused' warning in the situations when there is no way to use it. If it is unused in your code in doesn't mean that it is unused at all + what is the purpose of the `unused` API method? what if someone will use it, should he remove suppression? The nice [place](https://github.com/codenvy/che-core/blob/2aa05905269a75d358ac282260d927bdca232d77/platform-api/che-core-api-core/src/test/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequestTest.java#L72-L73) to use `@SuppressWarnings("unused")`